artemis/src/main/java/tech/pegasys/artemis/cli/options/LoggingOptions.java

```diff
@@ -81,7 +83,7 @@ public boolean isLogIncludeEventsEnabled() {
   }
 
   public String getLogDestination() {
-    return logDestination;
+    return logDestination.name();
```

Comment:
The code that uses this value expects the `key` not the enum `name()`.  Currently key is internal only so a getter would be needed. I think it happens to work out at the moment but we should actually refer to the same value.  I suspect the `key` value is no longer relevant and we should just use `.valueOf` to look up the matching value.  Or just pass `LoggingDestination` enum through instead of going to `String` and back if the class path will let us do that.

Reply:
the project dependencies are pretty ugly. i'll have another go.

Reply:
I wouldn't try adjusting dependencies to make it happen.  Just make sure we consistently use either name or key to go to string and back.